packages/form-js-editor/src/core/index.js

```diff
@@ -1 +1,13 @@
-export { default as FormEditorCore } from './FormEditorCore';
\ No newline at end of file
+import EventBus from 'diagram-js/lib/core/EventBus';
```

Comment:
Proposal: Provide own eventBus that simply exports the diagram-js event bus used here.

Allows us to import the eventbus from this project for type support :arrow_down:

Reply:
I'm not sure how to add types to the `EventBus` class outside of its implementation. 🤔

Reply:
I'll follow up on this :muscle: